uint8 role
uint32 id
float32 dis
float32 fp_rssi
float32 rx_rssi
